tests/features/import-explorer.feature

```diff
@@ -0,0 +1,135 @@
+Feature: Importer Explorer - View and configure importers on importers details page
+  Background: As a Devsecops I want to view and configure all of the Importers currently configured including default Importers
+
+#Scenario: The user can display importers page with all relevant field names
+Given TPA 2.0 is installed successfully
+When the user selects the importers menu option
+Then importers page is displayed showing the fields Name, Type, Description, Source, Period and Status in a table
+And there will be 3 dots (Hamburger icon) displayed for each importer
+
+#Scenario: The user can check pagination controls on importers page
+Given TPA 2.0 is installed successfully
+When the user selects the importers menu option
+Then the importers page will be displayed with pagination controls with correct values
+And it will be possible to move between pages accordingly
+And it will be possible to add page numbers in the pagination text box and pressing Enter to move to the desired page
```

Comment:
We should have generic pagination scenario and scripts to verify pagination for all pages on the application
```suggestion
Scenario: Pagination controls on importer explorer page
  When The user is on Importer Explorer page
  Then the importers page will be displayed with pagination controls with correct values
  And it will be possible to move between pages accordingly
  And it will be possible to add page numbers in the pagination text box and pressing Enter to move to the desired page
```

Reply:
Done